deployment/provider-azure-installer-windows.yaml

```diff
@@ -21,27 +21,39 @@ spec:
         app: csi-secrets-store-provider-azure
     spec:
       serviceAccountName: csi-secrets-store-provider-azure
+      hostNetwork: true
       containers:
         - name: provider-azure-installer
           image: mcr.microsoft.com/k8s/csi/secrets-store/provider-azure:0.0.8
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
+          args:
+            - --endpoint=unix://C:\\provider\\azure.sock
           resources:
             requests:
               cpu: 100m
-              memory: 200Mi
+              memory: 512Mi
             limits:
-              cpu: 100m
-              memory: 200Mi
+              cpu: 200m
+              memory: 1024Mi
           env:
             - name: TARGET_DIR
-              value: "C:\\k\\secrets-store-csi-providers"
+              value: "C:\\provider"
           volumeMounts:
-            - mountPath: "C:\\k\\secrets-store-csi-providers"
+            - mountPath: "C:\\provider"
               name: providervol
+            - name: mountpoint-dir
+              mountPath: "C:\\var\\lib\\kubelet\\pods"
+              mountPropagation: Bidirectional
```

Comment:
why do we need bidirectional now?

Reply:
https://kubernetes-csi.github.io/docs/deploying.html#driver-volume-mounts

```
The mount point used by the CSI driver must be set to Bidirectional to allow Kubelet on the host to see mounts created by the CSI driver container. See the example below:
```

If we don't set it to `Bidirectional` then the contents written to the target path will not show up in the pod mount.